app/dashboard/templates/onepager/send2.html

```diff
@@ -164,4 +168,12 @@ <h3>{% trans "with instructions about how to receive their tip." %}</h1>
     {% trans "We send the funds directly to the address associated with the user." %}</span>
   </div>
 </section>
+
+  {% if not user_json and username %}
+    <script>
+    setTimeout(function() {
+      alert("Sorry, we can find the user @{{ username }}");
```

Comment:
lets remove the `username` here - I'm not sure exactly how the django template injection prevention would work here, but I worry someone will create a URL with the username `"); do_something_bad();`

Reply:
Sure, no  problem. Just by reference https://docs.djangoproject.com/en/2.2/topics/security/#cross-site-scripting-xss-protection

Reply:
Yeah - I'm just paranoid :) I trust the Django devs are good and all, but you can never be too careful!